app/models/employer.rb

```diff
@@ -1,13 +1,13 @@
-require 'CSV'
-class Employer < ApplicationRecord
-
-	has_one :employer_csv
-	validates :name,  presence: true
-
-	def self.get_csv_layout
-		data = CSV.read("spec/fixtures/AcmeCo.csv")
-		Employer.employer_csv.create(Employer: employer, columns: data[0])
-	end
-
+# frozen_string_literal: true
 
+require 'csv'
+class Employer < ApplicationRecord
+  has_one :employer_csv_layout
+  has_many :emplyees
+  has_many :earnings
+  validates :name, presence: true
+
+  def create_earnings(file)
+    earnings.new.load_earnings(file)
```

Comment:
I'm not clear on this point.
I'm creating earnings for the employer, and not saving any data on the employer level.
In the current scenario, this could happen if we call it on the controller level with other actions but not on the model.